### DISCUSSION TITLE
Enable sorting on Alternative Identifiers (otherCatalogNumbers)

### DISCUSSION THREAD

**Discussion by ljwalker** [Symbiota/community-board#658] [Wish List]
Suggestion received during the December 2, 2024 Symbiota Support Group meeting.

Related suggestion received in chat:
> Being able to sort by identifier value would be super helpful for sorting barcode numbers for imaged specimens. Please make that possible some day

Related suggestion: Enable wildcard searching for alternative identifiers

**Comment by fbungartz** [comment]
This should be united with the request to be able to also search for identifier tags.
